src/key.rs

```diff
@@ -439,12 +682,31 @@ pub enum Action {
     #[serde(rename = "stats.get")]
     StatsGet,
     /// Provides access to the [create dump](https://docs.meilisearch.com/reference/api/dump.md#create-a-dump) endpoint. **Not restricted by `indexes`.**
-    #[serde(rename = "dumps.create")]
+    #[serde(rename = "dumps.execute")]
```

Comment:
Is this on the same PR on purpose?

Reply:
Yes! For me, since these are the `Actions` that you are able to determine in the `actions` of a `Key` It is more a change to the Key API than the dump api.